src/pages/PullRequestPage/subroute/FileExplorer/FileExplorer.jsx

```diff
@@ -49,7 +58,7 @@ function FileExplorer() {
       />
       {isLoading && <Loader />}
       {data?.length === 0 && !isLoading && (
-        <MissingFileData isSearching={isSearching} />
+        <MissingFileData isSearching={isSearching} hasFlagsSelected={!!flags} />
```

Comment:
Hrrm, thoughts on swapping this to `flags?.length > 0` currently it will return `true` if the list exists and nothing is selected.